db/seeds/05_projects.rb

```diff
@@ -425,3 +425,37 @@
 tmc_ops.subtitle = 'aplikacja do zarządzania hotelami'
 
 tmc_ops.save
+
+# Julinek AR
+julinek_ar = Project.where(slug: 'julinek-ar').first_or_initialize
+julinek_ar.assign_attributes(
+  display: true,
+  dofollow: false,
+  title: 'Julinek AR',
+  url: 'https://apps.apple.com/us/developer/grzegorz-lisowski/id1665804173',
```

Comment:
Not sure if this is useful in any way but julinekAR is also on android: https://play.google.com/store/apps/details?id=com.Trusense.JulinekWyspy&hl=pl

Reply:
Thanks, we may need to rethink our data structure in the future. Good to know, I took a note.